app/assets/javascripts/sufia/app.js

```diff
@@ -46,6 +47,16 @@ Sufia = {
 
     transfers: function () {
         $("#proxy_deposit_request_transfer_to").userSearch();
+    },
+
+    relationships_table: function () {
+        var rel = require('sufia/relationships_table');
+        $('table.relationships-api-enabled').each(function () {
+            var add_url = $(this).data('api-add-url');
```

Comment:
They referenced from $this.add_url, $this.remove_url, and $this.works_context used in the ajaxy part of the functionality.